### ISSUE TITLE
Redesign the footer along with contact section

### ISSUE THREAD

**Issue by Subhas6033** [xxRAJA/RAJA-PROFILE#44]
The footer section, along with the "Contact Us" section, needs a redesign to improve the visual appeal, accessibility, and usability. The current design is outdated and lacks a cohesive structure for easy navigation and information access

**Comment by Subhas6033** [comment]
@xxRAJA, I want to add this features. Please assign me under JWOC to go ahead.
